internal/interpreters/r.go

```diff
@@ -51,6 +51,7 @@ type RInterpreter interface {
 	GetPreferredPath() string
 	CreateLockfile(util.AbsolutePath) error
 	RenvEnvironmentErrorCheck() *types.AgentError
+	IsRenvInstalled(rexecPath string) *types.AgentError
```

Comment:
I exposed this because the `ScanDependencies` of  `LockfilePackageMapper` does not require a working R Library, it only needs the `renv`  package to be isntalled.